model-optimizer/extensions/ops/roll.py

```diff
@@ -49,3 +49,16 @@ def __init__(self, graph: Graph, attrs: dict):
 def roll_infer(node: Node):
     PermuteInputs().set_input_permutation(node.in_node(2), node, 'input:0', 'axis')
     copy_shape_infer(node)
+
+
+class TFRoll(Op):
```

Comment:
Why do we need this? I do not see extractor of `TFRoll` or any other mentions of this op.

Reply:
Fixed.